config/initializers/extensions.rb

```diff
@@ -0,0 +1,17 @@
+require 'delayed_job'
```

Comment:
I'd call this file `delayed_job` to be more semantic.

Reply:
Done.  OK to merge once the tests pass? @himal-mandalia ?